src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -163,6 +163,7 @@
       <Compile Include="$(BclSourcesRoot)\Mono\MonoListItem.cs" />
       <Compile Include="$(BclSourcesRoot)\Mono\MonoDomain.cs" />
       <Compile Include="$(BclSourcesRoot)\Mono\MonoDomainSetup.cs" />
+      <Compile Include="$(BclSourcesRoot)\Mono\MonoPInvokeCallbackAttribute.cs" />
```

Comment:
Please log issue to undo this change

Reply:
Implied by https://github.com/dotnet/runtime/issues/39100